File: programme/cube-of-number/cube_of_number.py
#!/usr/bin/env python3
# Write a programme to find cube of a number


def cube_of_number() -> None:
    """Returns the cube of a number"""
    n = int(input("Input  : "))
    print("Output :", pow(n, 3))


if __name__ == '__main__':
    cube_of_number()

File: programme/cube-of-number/CubeOfNumber.java
// Write a Java programme to find cube of a number

import java.util.Scanner;

class CubeOfNumber {
  
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Input: ");
    double input = sc.nextDouble();
    System.out.println("Output: "+cube(input));
  }
  
  private static double cube(double num) {
    return Math.pow(num, 3);
  }
}
